src/components/ContactUs/Form.js

```diff
@@ -17,10 +18,12 @@ function Form() {
       )
       .then(
         (result) => {
-          alert(result.text);
+          <Notification data={result} />;
+          // eslint-disable-next-line no-console
+          console.log('notification is sent!');
```

Comment:
Please remove the console.log and the eslint exception. I believe you added the console log just for testing but the problem is, I cant see the notification after I submit my message, I only can see the console.log.

Reply:
It's done! I installed `react-toastify` library to get notification when the form is submitted.